assets/js/base/components/cart-checkout/totals/shipping/shipping-placeholder.tsx

```diff
@@ -39,6 +39,10 @@ export const ShippingPlaceholder = ( {
 
 	return (
 		<CalculatorButton
+			label={ __(
+				'Add an address for shipping rates',
```

Comment:
This uses the term "shipping rates", yet the message below and in other places uses "shipping options". We should be consistent because they both mean the same thing.

Reply:
Right, we should keep the text the same everywhere. I have updated the label. Thank you! 👍